Gemfile

```
@@ -11,7 +11,7 @@ gem 'ancestry', '~> 4.0'
 gem 'scoped_search', '>= 4.1.10', '< 5'
 gem 'ldap_fluff', '>= 0.5.0', '< 1.0'
 gem 'apipie-rails', '>= 0.8.0', '< 2'
-gem 'apipie-dsl', '>= 2.2.6'
+gem 'apipie-dsl', '>= 2.6.0'
```

This should at least require 2.6.1
```suggestion
gem 'apipie-dsl', '>= 2.6.1'
```